operator/validator/controller.go

```diff
@@ -967,8 +957,14 @@ func (c *controller) onShareInit(share *ssvtypes.SSVShare) (*validators.Validato
 		c.printShare(v.Share(), "get validator")
 	}
 
+	var vc *validator.Committee
+
 	// Start a committee validator.
-	vc, found := c.validatorsMap.GetCommittee(operator.CommitteeID)
+	found = c.validatorsMap.UpdateCommitteeAtomic(operator.CommitteeID, func(fvc *validator.Committee) {
+		fvc.AddShare(&share.Share)
+		vc = fvc
+	})
```

Comment:
Would be cleaner to return `validator.Committee` from `UpdateCommitteeAtomic`, wdyt ?

Reply:
We could, but there are 3 call sites to this function and only this one would benefit from the extra return value. @y0sher @MatusKysel your take here?